testing/values.go

```diff
@@ -22,6 +22,8 @@ const (
 	FirstChannelID    = "channel-0"
 	FirstConnectionID = "connection-0"
 
+	SecondClientID = "07-tendermint-1"
```

Comment:
now that solomachine clients are in the mix, what do you think about instead of this constant, we have a function like

```go
secondClientID := TendermintClientID(2)
```

Reply:
I like this idea!